lib/vnet/daemon/service_darwin.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/gravitational/trace"
 
 	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/utils/darwinbundle"
```

Comment:
`darwinbundle` wasn't backported to v17, so this probably can just be removed.

idk if this not being backported is going to cause issues for you, but the refactor was mostly about extracting things out of `lib/vnet/daemon` that I needed for os_log integration, so I think we should be fine.